template.yaml

```diff
@@ -16,11 +16,17 @@ spec:
         - component
         - owner
         - slack_contact_channel
+        - github_topic
       properties:
         product:
           title: Product
           type: string
           description: Product this component belongs to, normally the team name, e.g. cmc
+        github_topics:
```

Comment:
naming looks out of sync

github_topic in required params, but github_topics elsewhere